jupyterhub/notebook-images/overlays/additional/minimal-notebook-imagestream.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     local: true
   tags:
   - annotations:
-      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8.3"}]'
+      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8.6"}]'
```

Comment:
`s2i-minimal-py38-notebook:v0.0.14` is python version `3.8.3`. I just did a quick check and there is a `v0.0.15` tag of the notebook that is version `3.8.6`.

Reply:
Thanks. I will revert this